Source/EasyNetQ.Tests/MultipleExchangeTest/MultipleExchangePublishExchangeDeclareStrategyTests.cs

```diff
@@ -34,7 +34,7 @@ public void When_declaring_exchanges_for_message_type_that_has_no_interface_one_
 
             var publishExchangeStrategy = new MultipleExchangePublishExchangeDeclareStrategy(conventions, advancedBus);
 
-            publishExchangeStrategy.DeclareExchangeAsync(typeof(MyMessage), ExchangeType.Topic).Wait();
+            publishExchangeStrategy.DeclareExchange(typeof(MyMessage), ExchangeType.Topic);
```

Comment:
Thanks!

It's a extensions which I made to fix compilation and we definitely should remove them.

Reply:
@zidad Is it ok for you if I move this extensions to test project? It makes sense for me, because there are a lot of tests there we have to wait synchronously and these extensions help to reduce boilerplate waiting code.

Reply:
Ah yes then sure! I'd prefer not to have sync extension methods at all, maybe even move all of them to a separate nuget package in a next version :)

Reply:
@zidad Deal :wink:

Actually, there is one more usage of these extensions in `Rpc`, but I don't want to touch `Rpc` in this PR, because of too much changes.

Imho, it's better to create several more issues and fix them independently(but before release of v4), because current PR is too big.